webapp/components/Dropdown.vue

```diff
@@ -113,6 +114,7 @@ export default {
       }
     },
     popoveMouseLeave() {
+      if (this.developperNoAutoClosing) return
```

Comment:
can't we use the `DEBUG` flag for this?

Reply:
Even in debugging it shall behave normal.
We need this only in very rear circumstances of debugging.